index.bs

```diff
@@ -2725,6 +2769,284 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 1. Return [=success=] with data null.
 
+</div>
+#### The browsingContext.locateNodes Command ####  {#command-browsingContext-locateNodes}
```

Comment:
feel-free-to-ignore suggestion: it might make it easier to review if we start with a single locator and then extend the list in PR

Reply:
Probably, but the decision was to first enable the locators defined by Classic, so that's what I was attempting to do. If it makes it easier, I can remove the others, and we can go from there, I suppose, but I have no idea if that really makes sense. The text would simply have to be replaced once one locator was agreed on.

Reply:
I think keeping the PR "as is" is fine, and will help us avoid churn